analyzers/tests/SonarAnalyzer.Test/TestCases/SecureRandomSeedsShouldNotBePredictable.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System;
```

Comment:
I think this is very well tested, maybe too well since the symbolic execution engine has it's own tests.

I did not see any tests with inheritance. Would it be worth adding some? `SecureRandom` is not sealed, although most of the methods we check are static so I'm not sure if it's actually useful.

Reply:
I will add a test on part 3, as in part1+2 I do not call `new` on SecureRandom, only `GetInstance`, which is a static method that _always_ returns `SecureRandom`.